azure_ad_admin/plugin.spec.yaml

```
@@ -471,6 +471,51 @@ actions:
         description: Was operation successful
         type: boolean
         required: false
+  update_user_info:
+    title: Update User Information
+    description: Update a users information
+    input:
+      city:
+        title: City
+        type: string
+        description: The city in which the user is located
+        required: false
```

Can you add an example key for each of these inputs with valid/working examples where possible and then regenerate on icon-plugin 4.8.4.

```suggestion
        required: false
        example: Boston
```